templates/web_questions/templates.yaml

```diff
@@ -0,0 +1,45 @@
+dataset: web_questions
+templates:
+  9b42df9f-8cca-4139-b61e-b1a4d2127e9d: !Template
+    id: 9b42df9f-8cca-4139-b61e-b1a4d2127e9d
+    jinja: 'If {% if answers|length > 1 %}"{{ answers[0:-1]|join(''", "'') }}" and
+      "{{ answers[-1] }}" are answers{% else %}"{{answers[0]}}" is an answer{% endif
+      %}, what is the question? ||| {{ question|capitalize }}
+
+      '
+    name: if-answers-what-question
+    reference: ''
+  9f4cd4a4-79e5-40b2-bb0d-f9a86396511a: !Template
+    id: 9f4cd4a4-79e5-40b2-bb0d-f9a86396511a
+    jinja: '{% set rand_val = range(0,answers|length)|choice %}
+
+      Give me a possible correct answer to the question "{{ question }}" ||| {{ answers[rand_val]
+      }}'
+    name: potential-correct-answer
+    reference: ''
+    task_template: false
+  ac5c940c-78eb-4f54-9426-e9e3a0304972: !Template
+    id: ac5c940c-78eb-4f54-9426-e9e3a0304972
+    jinja: What would be a credible question corresponding to the {% if answers|length
+      == 1 %}answer "{{ answers[0] }}"{% else %}answers "{{ answers[0:-1]|join('",
+      "') }}" and "{{ answers[-1] }}"{% endif %}? ||| {{ question|capitalize }}
+    name: credible-question
+    reference: ''
+  d4b38133-23b6-402f-90dc-8b0acf7963ca: !Template
+    id: d4b38133-23b6-402f-90dc-8b0acf7963ca
+    jinja: 'So I''ve got {% if answers|length > 1 %}{{ answers|length}} possible answers:
+      "{{ answers[0:-1]|join(''", "'') }}" and "{{ answers[-1] }}"{% else %} 1 possible
+      answer: "{{answers[0]}}"{% endif %}. Can you suggest a question? ||| {{ question|capitalize
+      }}'
+    name: suggest-question
+    reference: ''
+    task_template: false
+  d5523864-cc61-4d1b-ac67-80605dc59640: !Template
+    id: d5523864-cc61-4d1b-ac67-80605dc59640
+    jinja: 'The question is "{{ question }}" and the possible answers are: {% if answers|length
+      > 1 %}"{{ answers[0:-1]|join(''", "'') }}" and "{{ answers[-1] }}"{% else %}"{{answers[0]}}"{%
+      endif %}. How many answers are there? ||| {{ answers|length }}
```

Comment:
I like that you're coming up with new tasks, but I think this one might be too easy.

Reply:
Shall I remove it or try to make it more complex somehow (for example reformulating as "I think it might be" instead of "the possible answers are")?

Reply:
I sort of feel like you are either listing all the answers, or you aren't. If you list all the answers, it's just counting, and if you don't list all the answers, it's kind of a total guess (since there are a crazy number of ways to state someone's name, for example).

Reply:
I might have been unclear, but I wasn't suggesting not listing all the answers, which of course would just be impossible. I was suggesting making the answer less straightforward by not having the repetition of "answers" in "the possible answers" and "how many answers are there". I agree counting should be easy (at least for small numbers) but then it could be interesting to see if the model can do something we expect it to find easy to do

Reply:
Sorry, wasn't saying you were suggesting to not include the list of answers - was just saying that wasn't an option. The worry about too-easy tasks is that fundamentally we only have a limited amount of training budget available, so it might be better spent training on task with more learning signal.

Reply:
No worries :-) Has anyone else include prompts like this? If not, it could be interesting to have a couple of prompts that test things like this (things that we think should be easy, but could benefit from being demonstrated), given the large number of prompts overall. I've reformulated it to make it slightly trickier - but can just remove it otherwise.